drivers/i2c/i2c_npcx_controller.c

```diff
@@ -184,6 +203,38 @@ static const struct npcx_i2c_timing_cfg npcx_20m_speed_confs[] = {
 	[NPCX_I2C_BUS_SPEED_1MHZ] = {.HLDT  = 7, .k1 = 16, .k2 = 10},
 };
 
+#if defined(CONFIG_PM) && defined(CONFIG_I2C_TARGET)
+static void i2c_npcx_pm_policy_state_lock_get(const struct device *dev,
+					      enum i2c_pm_policy_state_flag flag)
+{
+	const struct i2c_ctrl_config *const config = dev->config;
+	struct i2c_ctrl_data *const data = dev->data;
+
+	if (config->wakeup_source == false) {
```

Comment:
nit: it's a boolean, just ` 	if (!config->wakeup_source) {`

Reply:
Done